soft4pes/control/mpc/solvers/__init__.py

```diff
@@ -1,7 +1,12 @@
-from soft4pes.control.mpc.solvers.mpc_enum import MpcEnum
+"""
+Solvers for model predictive control algorithms.
```

Comment:
in general, whenever we write "model predictive control" let's have also after that "(MPC)". Alternatively, we can just have "MPC". Maybe the first option --> "model predictive control (MPC" is better as it has better readability

Reply:
Fixed in the next commit